### PR TITLE
Add signup step

### LINKED ISSUE #124: Include "join us" flow at the end

### PR DESCRIPTION
**Pull request by jacknugent**
Closes  #124

### CODE REVIEW

**Comment by jacknugent** [comment]
image

File: frontend/src/Pages/TakeAction/ScrollSpy/ScrollSpy.tsx [hunk 1]
```diff
@@ -75,11 +82,19 @@ export default function ScrollSpy({ isEmailSent, isPhoneCallMade, isSocialPosted
                     step={4}
                     id="#post_on_social"
                     state={linkState(isSocialPosted, isPhoneCallMade)}
-                    last
                     desktop={desktop}
                 >
                     {desktop ? "Post on Social" : "Post"}
                 </Step>
+                <Step
```
Comment (kevincianfarini) [inline frontend/src/Pages/TakeAction/ScrollSpy/ScrollSpy.tsx:89]: I think we should only conditionally show this step. If a call to check if the current user is already a changemaker returns true, we should omit this step.

Comment (jacknugent) [reply frontend/src/Pages/TakeAction/ScrollSpy/ScrollSpy.tsx:89]: Is there currently a call to check if the user is a changemaker?

Comment (kevincianfarini) [reply frontend/src/Pages/TakeAction/ScrollSpy/ScrollSpy.tsx:89]: Yes! A call to `/check-membership` will do it. Right now it's just mocked out to always return false. 

See more here https://github.com/climatechangemakers/climate-changemakers-act/pull/157

Comment (jacknugent) [reply frontend/src/Pages/TakeAction/ScrollSpy/ScrollSpy.tsx]: Fabulous, added

**Pull request merged**